index.js

```diff
@@ -1,7 +1,30 @@
 // Iteration 1: Names and Input
 
+var hacker1 = { firstName: 'Jeremy', lastName: 'Couderc' };
+var message = 'The drivers name is ' + hacker1.firstName + ' ' + hacker1.lastName + '!!';
+console.log(message);
 
 // Iteration 2: Conditionals
 
+const message = `The driver has the longest name, it has XX characters`;
```

Comment:
Because you already have a message previously we wouldn't try to redeclare it again